src/game/mario.h

```diff
@@ -6,6 +6,11 @@
 #include "macros.h"
 #include "types.h"
 
+// Continues running action logic, e.g. after Mario's action changes
+#define ACTION_CONTINUE TRUE
+// Finishes running action logic for the current frame
+#define ACTION_FINISH FALSE
+
```

Comment:
This seems a strange place to put this tbh. Would it not make more sense together with the other mario action defines?

Reply:
addressed